modules/rostests/apitests/user32/PrivateExtractIcons.c

```diff
@@ -90,6 +91,12 @@ static struct
 
     /* Icon group file containing one PNG icon and one normal icon */
     {L"%temp%\\ROS.ico", 1, 1, TRUE},
+
+    /* Executable file with bad 'Icon Group' but good 'Icons'.
+     * Windows explorer shows the program's icon correctly in WinXP/Win2K3
+     * but Windows 7 shows only a default icon. This is analogous
+     * to EXE's generated by older Watcom C/C++ versions. */
```

Comment:
```suggestion
     * to EXE's generated by older Watcom C/C++ versions */
```

Reply:
The period here is not something that will add to a 'debug.log', so it should be OK. Thanks.